tests/benchmarks/multicore/idle_with_pwm/Kconfig

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config TEST_SLEEP_DURATION_MS
+	int "Amount of time (in miliseconds) the core is sleeping"
+	default 1000
+	help
+		Set sleep duration to TEST_SLEEP_DURATION_MS miliseconds.
+		Based on the value of 'min-residency-us' specified for each power state defined in the DTS,
```

Comment:
min-residency-us is defined in DTS:
https://github.com/nrfconnect/sdk-zephyr/blob/main/dts/common/nordic/nrf54h20.dtsi#L127

Core has to sleep for more than 'min-residency-us' in order to enter that specific low power state.
It is used in main.c:
https://github.com/nordic-segl/sdk-nrf/blob/NRFX-6370_PWM_with_s2ram/tests/benchmarks/multicore/idle_with_pwm/src/main.c#L93

Default value of 1000ms is greater than min-residency for S2RAM state.

Here
https://github.com/nordic-segl/sdk-nrf/blob/NRFX-6370_PWM_with_s2ram/tests/benchmarks/multicore/idle_with_pwm/testcase.yaml#L28
KConfig value is set to 500 ms, which is too low to enter the deepest S2RAM while sufficient to enter IDLE.

Reply:
Use a multiplier then and in the code take the dts property and multiply it so it can work with any board/future soc?

Reply:
I think whatever I do it will not scale easily. Power states and min-residency-us is very specific to a target (in fact, core).

"in the code take the dts property"
1. I can't take dts property of &s2ram as it is valid only for nrf54h20dk/nrf54h20/cpuapp.
2. I can't take dts property of &idle because it's going to be removed:
https://github.com/nrfconnect/sdk-zephyr/pull/2032/files#diff-3455660ed5585b71497d4674c6a03a570690d462406141ea0170df7c10f28391